cpp/CMake/Modules/ConfigureGlog.cmake

```diff
@@ -60,7 +58,7 @@ set(GLOG_INCLUDE_DIR "${GLOG_ROOT}/install/include")
 set(GLOG_FOUND TRUE)
 
 IF(WIN32)
-        set(GLOG_LIBRARIES ${GLOG_INSTALL}/lib/glogd.lib)       
+        set(GLOG_LIBRARIES ${GLOG_INSTALL}/lib/glogd.lib)
```

Comment:
Lets use te glog.lib and not the debug one. These are from the previous pull request. But lets fix those here